src/pki.rs

```diff
@@ -56,15 +58,15 @@ impl Keyblob {
         let mut encrypted_keyblob = [0; 0xB0];
         encrypted_keyblob[0x20..].copy_from_slice(&self.0);
 
-        let mut crypter = Ctr128::<Aes128, ZeroPadding>::new_fixkey(
+        let mut crypter = Ctr128::<Aes128>::new(
             GenericArray::from_slice(&key.0),
             GenericArray::from_slice(&encrypted_keyblob[0x10..0x20]),
         );
-        crypter.encrypt_nopad(&mut encrypted_keyblob[0x20..])?;
+        crypter.apply_keystream(&mut encrypted_keyblob[0x20..]);
```

Comment:
Not sure if it's the right thing to do here.

Reply:
Yeah this is correct. Aes CTR is a stream cipher, there never was any padding to begin with!